panic: runtime error: invalid memory address or nil pointer dereference

Hello, I am getting this error after building the binary and trying to run it:

```
 ./netapp-api-exporter -c /Users/USER/git/personal/netapp-api-exporter/netapp_filers.yaml -w 100
2019-08-29 09:49:16.550 [info] Host (IP) loaded	
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd8 pc=0x1486d72]

goroutine 35 [running]:
main.(*Filer).getVolumeList(0xc0001605a0, 0xc0001aa030, 0x0, 0x0, 0x0)
	/Users/challak/go-workspace/src/github.com/sapcc/netapp-api-exporter/volume.go:127 +0x52
main.(*Filer).GetNetappVolume(0xc0001605a0, 0xc0001c4000, 0xc0001c4060)
	/Users/challak/go-workspace/src/github.com/sapcc/netapp-api-exporter/volume.go:62 +0x33a
main.main.func1(0xc0001c4000, 0xc0001c4060, 0xc0001c40c0, 0xc0001c4120, 0x1670820, 0xc000188400)
	/Users/challak/go-workspace/src/github.com/sapcc/netapp-api-exporter/main.go:68 +0x4e
created by main.main
	/Users/challak/go-workspace/src/github.com/sapcc/netapp-api-exporter/main.go:60 +0x2e3

```

Config file:

```
- name: ntap1
  host: HOST_IP
  username: admin
  password: Netapp123

```

I have verified the credentials work but I am at a loss on why the panic happens.

--- a/main.go
+++ b/main.go
@@ -130,7 +130,7 @@ func loadFilerFromFile(fileName string) (c []*Filer) {
 		logger.Fatal("[ERROR] ", err)
 	}
 	for _, b := range fb {
-		c = append(c, &Filer{FilerBase: b})
+		c = append(c, NewFiler(b.Name, b.Host, b.Username, b.Password, b.AvailabilityZone))
 	}
 	return
 }

